### DISCUSSION TITLE
Host Adonis app on HostGator

### DISCUSSION THREAD

**Discussion by thethirdandres** [adonisjs/core#1569] [Help]
I can't seem to find any tutorials that teach this. All of them only show deploying using Heroku. Do any of you have references I can check out?

**Comment by brandonsousa** [comment]
You perform the normal deploy, you have to have the node installed in the vps and your project cloned in it, then run the "node server.js" inside the directory, but the most recommended is to use something with pm2, in case the application or the server stops it automatically restarts

**Comment by miqbalrr** [comment]
if you are using a hosting service, please make sure that the hosting provider compatible with nodejs

**Comment by brandonsousa** [comment]
does your server support nodejs?
your server needs to be a vps to be able to use nodejs since the shared server only runs php.
